code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -27,12 +27,31 @@
 	gold_core_spawnable = 1
 	faction = list("zombie")
 	var/mob/living/carbon/human/stored_corpse = null
+	var/obj/item/organ/tongue/zombie/stored_tongue = null
 	butcher_results = list(/obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/zombie = 3)
 	see_invisible = SEE_INVISIBLE_MINIMUM
 	see_in_dark = 8
 	layer = MOB_LAYER - 0.1
 	var/removingairlock = 0
 
+	var/they_are_just_sleeping_msg = "<span class='userdanger'>They'll be \
+		getting up on their own, just give them a minute!</span>"
+	var/down_but_not_out_msg = "<span class='userdanger'>You're down, but \
+		not quite out. You'll be back on your feet within a minute \
+		or two.</span>"
```

Comment:
Each of these are only used in a single proc, you don't need to make those vars for the mob, just put those lines at the beginning of the zombie/AttackingTarget() and zombie/death() below.